content/03.ml-concepts.md

```diff
@@ -3,6 +3,7 @@ Deep learning is a distinct subfield of machine learning, but it is still a subf
 DL has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches in certain contexts, but it is not immune to the many limitations inherent to machine learning.
 Many best practices for machine learning also apply to deep learning.
 For instance, deep supervised learning models should be trained, tuned, and tested on non-overlapping datasets.
+Similarly, to prevent overfitting, the data used for testing should be locked and only used one-time for evaluating the very final tuned model.
```

Comment:
Very important point! Maybe we could rephrase " very final tuned model" to 

> ... the final model after all tuning steps were completed. Using a test set more than once will lead to biased estimates of the generalization performance.

Reply:
Done